homeassistant/components/bkk/sensor.py

```diff
@@ -0,0 +1,311 @@
+"""BKK Futar integration to monitor public transport stops of Budapest city."""
+
+from datetime import datetime, timedelta
+import logging
+
+import futar
+import voluptuous as vol
+
+from homeassistant.components import group
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import EVENT_HOMEASSISTANT_START, STATE_UNAVAILABLE
+from homeassistant.helpers import entity_registry
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+DOMAIN = "bkk"
+CONF_STOP = "stop"
+CONF_GROUPED = "grouped"
+CONF_ONLY_DEPARTURES = "only_departures"
+CONF_MINUTES_AFTER = "minutes_after"
+MAX_DEPARTURES = 5
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_STOP): cv.string,
+        vol.Optional(CONF_GROUPED, default=False): cv.boolean,
+        vol.Optional(CONF_ONLY_DEPARTURES, default=True): cv.boolean,
+        vol.Optional(CONF_MINUTES_AFTER, default=120): cv.positive_int,
+    }
+)
+
+SCAN_INTERVAL = timedelta(seconds=30)
+
+ICONS = {
+    "BUS": "mdi:bus",
+    "TRAM": "mdi:tram-side",
+    "RAIL": "mdi:tram",
+    "SUBWAY": "mdi:subway",
+    "API": "mdi:api",
+}
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_create_group(hass, name, entities):
```

Comment:
Groups are only allowed to be created by a user and not by integrations.

Reply:
Why? I just wanted to make it easier grouping the departures belonging to a stop.

Reply:
Groups are only meant as a user interface for grouping entities. Not an interface for integrations to group entities. We don't want integrations to group entities by default.